packages/importer/src/providers/standardnotes/types.ts

```diff
@@ -0,0 +1,75 @@
+
+export enum ContentTypes {
+  Note = "Note",
+  Tag = "Tag",
+  SmartTag = "SN|SmartTag",
+  Component = "SN|Component" 
+}
+
+export type EditorType = {
+  type: "code" | "html" | "text" | "json" | "markdown";
+  jsonFormat?: "table" | "token";
+  mode?: string;
```

Comment:
We can do with a better name here. `mode` is used mainly for providing the language for the code block right? Perhaps we should name it `language` or something.